recipe.js

```diff
@@ -226,4 +226,18 @@ function getRandomRecipeById() {
 
 
 }
-getRandomRecipeById();
\ No newline at end of file
+getRandomRecipeById();
+
+//Select multiple options
```

Comment:
I don't think this code is useful here since the selection would have to happen on the index page instead of the recipe page